hrm-domain/hrm-core/permission-rules/th.json

```diff
@@ -3,19 +3,21 @@
   "closeCase":  [["isSupervisor"]],
   "reopenCase":  [["isSupervisor"]],
   "caseStatusTransition":  [["isSupervisor"]],
-  "addNote":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
-  "editNote":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
-  "addReferral":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
-  "editReferral":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
-  "addHousehold":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
-  "editHousehold":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
-  "addPerpetrator":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
-  "editPerpetrator":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
-  "addIncident":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
-  "editIncident":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
-  "addDocument":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
-  "editDocument":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
-  "editCaseOverview":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+  "addNote":  [["isSupervisor"],["isCreator", "isCaseOpen"], ["isCaseContactOwner", "isCaseOpen"]],
+  "editNote":  [["isSupervisor"],["isCreator", "isCaseOpen"], ["isCaseContactOwner", "isCaseOpen"]],
+  "addReferral":  [["isSupervisor"],["isCreator", "isCaseOpen"], ["isCaseContactOwner", "isCaseOpen"]],
+  "editReferral":  [["isSupervisor"],["isCreator", "isCaseOpen"], ["isCaseContactOwner", "isCaseOpen"]],
+  "addHousehold":  [["isSupervisor"],["isCreator", "isCaseOpen"], ["isCaseContactOwner", "isCaseOpen"]],
+  "editHousehold":  [["isSupervisor"],["isCreator", "isCaseOpen"], ["isCaseContactOwner", "isCaseOpen"]],
+  "addPerpetrator":  [["isSupervisor"],["isCreator", "isCaseOpen"], ["isCaseContactOwner", "isCaseOpen"]],
+  "editPerpetrator":  [["isSupervisor"],["isCreator", "isCaseOpen"], ["isCaseContactOwner", "isCaseOpen"]],
+  "addIncident":  [["isSupervisor"],["isCreator", "isCaseOpen"], ["isCaseContactOwner", "isCaseOpen"]],
+  "editIncident":  [["isSupervisor"],["isCreator", "isCaseOpen"], ["isCaseContactOwner", "isCaseOpen"]],
+  "addDocument":  [["isSupervisor"],["isCreator", "isCaseOpen"], ["isCaseContactOwner", "isCaseOpen"]],
+  "editDocument":  [["isSupervisor"],["isCreator", "isCaseOpen"], ["isCaseContactOwner", "isCaseOpen"]],
+  "editCaseOverview":  [["isSupervisor"],["isCreator", "isCaseOpen"], ["isCaseContactOwner", "isCaseOpen"]],
+  "editChildIsAtRisk":  [["isSupervisor"],["isCreator", "isCaseOpen"], ["isCaseContactOwner", "isCaseOpen"]],
```

Comment:
I think editChildIsAtRisk and editFollowUpDate are no longer needed. 
@stephenhand  can confirm.
Besides that it lgtm, but I've been told that this repo needs explicit approval from an engineer :)

Reply:
Yes those 2 are redundant and should be removed. We should probably just do a sweep and remove them from all accounts in both projects, to prevent them keep popping up again